cfgmgr/buffer_headroom_mellanox.lua

```diff
@@ -98,7 +99,14 @@ local default_lossless_param_keys = redis.call('KEYS', 'DEFAULT_LOSSLESS_BUFFER_
 local over_subscribe_ratio = tonumber(redis.call('HGET', default_lossless_param_keys[1], 'over_subscribe_ratio'))
 
 -- Fetch the shared headroom pool size
-local shp_size = tonumber(redis.call('HGET', 'BUFFER_POOL|ingress_lossless_pool', 'xoff'))
+-- POC, HUA: Fetch from both config DB and static config DB.
+local shp_size_val = redis.call('HGET', 'BUFFER_POOL|ingress_lossless_pool', 'xoff')
+if not shp_size_val then
```

Comment:
There can be two cases which fails hget:
1. The entry doesn’t exist, we can fallback to profile db in this case 
2. The entry exists but no xoff field. In this case, we do not need to check profile db, right?

Or each entry must have all the fields so the 2nd case can’t happen?
Maybe we discussed it before, just to confirm